cart/module.go

```diff
@@ -158,6 +158,10 @@ func (r *routes) apiRoutes(registry *web.RouterRegistry) {
 	registry.HandlePost("cart.api.applyVoucher", r.apiController.ApplyVoucherAndGetAction)
 	registry.HandlePut("cart.api.applyVoucher", r.apiController.ApplyVoucherAndGetAction)
 
+	registry.Route("/api/cart/removevoucher", `cart.api.removeVoucher(couponCode)`)
+	registry.HandlePost("cart.api.removevoucher", r.apiController.RemoveVoucherAndGetAction)
+	registry.HandleDelete("cart.api.removevoucher", r.apiController.RemoveVoucherAndGetAction)
+
```

Comment:
cart.api.removeVoucher is declared in camel case but used in lower case

cart.ape.removevoucher. If an exact match is necessary, please adjust

Reply:
I updated the handle name accordingly.